docs/intro/installation.rst

```diff
@@ -4,84 +4,41 @@
 Installation
 ************
 
-The SunPy project `maintains a range of packages <https://sunpy.org/project/affiliated>`__ that leverage the wider ecosystem of scientific Python packages for solar physics.
+This is the first chapter in the sunpy tutorial, and by the end of it you should have a working installation of Python and sunpy.
 
 Installing Python
 =================
-These instructions will set you up with `miniforge <https://conda-forge.org/docs/user/introduction.html>`__, which makes it easy to install and manage Python packages.
+There are many ways to install Python, but even if you have Python installed somewhere on your computer we recommend following these instructions anyway.
+That's because we will create a new Python environment.
+As well as containing a Python installation, this environment provides an isolated place to install Python packages (like sunpy).
 
-To install the miniforge Python distribution follow `the miniforge installation instructions <https://github.com/conda-forge/miniforge#install>`__.
+The package manager we'll be using is called ``conda``.
+To install conda, follow the `the miniforge installation instructions <https://github.com/conda-forge/miniforge#install>`__.
+This will install ``conda`` and automatically configure the default channel (a channel is a remote software repository) to be ``conda-forge``, which is where ``sunpy`` packages are available.
```

Comment:
```suggestion
This will install ``conda`` and automatically configure the default channel (a channel is a remote software repository) to be ``conda-forge``, which is where ``sunpy`` package and the other SunPy packages are available.
```
Kind of confusing what I wrote.

Reply:
We don't actually mandate that affialiated packages are on conda forge at the moment, so I don't want to add this. (FWIW I'd be in favour of mandating affiated pacakges are on conda-forge)